### DISCUSSION TITLE
原，铁，零玩家聚集地！

### DISCUSSION THREAD

**Discussion by zhukeyu666** [share121/inter-knot#2225] [其他]
欢迎各位旅行者/开拓者/绳匠的到来！群内聊天，配队，交友都可。欢迎加入
Q群：663851293
![qrcode_1723715026408](https://github.com/user-attachments/assets/586c05b7-fc9d-42d6-8050-de95492fd9ff)

**Comment by YuTinMIn** [comment]
这群正经吗？

**Comment by Dave-12138** [comment]
鹅群我有四不加
正经的不加，因为没意思
<img src="https://bbs-static.miyoushe.com/static/2023/05/31/d3aff69a7813091e4bbab00976c01102_8986465711411928814.png" alt="希儿 哈欠" width="75" >
不正经的不加，因为nsfw
<img src="https://bbs-static.miyoushe.com/static/2023/12/22/c9b0cbc0bd794d49e5db8a0f446d91c3_6491084927481570674.png" alt="月下-当场抓走" width="75" >
键政的不加，因为群很快就没了
<img src="https://bbs-static.miyoushe.com/static/2023/12/19/2f3dbb500f3e7f78d9c32f577a155c62_2550392836154570910.png" alt="雪衣 逮捕" width="75" >
不键政的不加，因为不键政说明有别的话题挤占了生态位，比如运粪车
<img src="https://upload-bbs.miyoushe.com/upload/2023/01/17/6a055646a2017fd16251feb4a06786a8_3524208470425464936.png" alt="无语呜呜" width="75" >

**Comment by nina-mmk** [comment]
现在真有萌新马
